docs/spfx/web-parts/guidance/simplify-adding-web-parts-with-preconfigured-entries.md

```diff
@@ -2,9 +2,9 @@
 
 > **Note:** The SharePoint Framework is currently in preview and is subject to change. SharePoint Framework client-side web parts are not currently supported for use in production environments.
 
-The added value of using web parts above other types of SharePoint customizations, is the ability for users to configure them to their specific needs. Using configuration properties exposed by developers, users can change how the particular web part behaves. The more configuration properties the web part exposes the more flexible it is. But all that configuration requires additional work from users after adding a web part to the page. In this article you will learn how you can use SharePoint Framework client-side web parts preconfigured entries to provide users with preconfigured versions of your web part.
```

Comment:
I worked on a new intro in which I was trying to make simpler, but what would complete it is an example, like "For example, a SharePoint web part that creates workflows may have multiple template types..." If you can think of a good example to add that would be great. If you like the original writing you can change this back.